api/shape/toolpath.js

```diff
@@ -6,21 +6,39 @@ import { computeToolpath } from '@jsxcad/geometry';
 import { lerp } from '@jsxcad/math-vec3';
 
 export const toolpath =
-  ({ diameter = 1, jumpHeight = 1 } = {}) =>
+  ({
+    diameter = 1,
+    jumpHeight = 1,
+    stepCost = diameter * -2,
+    turnCost = -2,
+    neighborCost = -2,
+    stopCost = 30,
```

Comment:
This is cool! So tuning these will change what it prioritizes?

Reply:
Yeah, although I'm not sure how much practical flexibility there will be.